requirements/doc.txt

```diff
@@ -0,0 +1,41 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file requirements/doc.txt requirements/base.in requirements/doc.in
+#
+alabaster==0.7.11         # via sphinx
+babel==2.6.0              # via sphinx
+bleach==2.1.3             # via readme-renderer
+certifi==2018.4.16        # via requests
+cffi==1.11.5              # via cmarkgfm
+chardet==3.0.4            # via doc8, requests
+cmarkgfm==0.4.2           # via readme-renderer
+django==1.11.14
+djangorestframework==3.6.4
+doc8==0.8.0
+docutils==0.14            # via doc8, readme-renderer, restructuredtext-lint, sphinx
+edx-sphinx-theme==1.3.0
+future==0.16.0            # via readme-renderer
+html5lib==1.0.1           # via bleach
+idna==2.7                 # via requests
+imagesize==1.0.0          # via sphinx
+jinja2==2.10              # via sphinx
+markupsafe==1.0           # via jinja2
+packaging==17.1           # via sphinx
+pbr==4.2.0                # via stevedore
+pycparser==2.18           # via cffi
+pygments==2.2.0           # via readme-renderer, sphinx
+pyparsing==2.2.0          # via packaging
+pytz==2018.5              # via babel, django
+readme-renderer==21.0
+requests==2.19.1          # via sphinx
+restructuredtext-lint==1.1.3  # via doc8
+six==1.11.0               # via bleach, doc8, edx-sphinx-theme, html5lib, packaging, readme-renderer, sphinx, stevedore
+snowballstemmer==1.2.1    # via sphinx
+sphinx==1.7.6
+sphinxcontrib-websupport==1.1.0  # via sphinx
+stevedore==1.29.0         # via doc8
+typing==3.6.4             # via sphinx
+urllib3==1.23             # via requests
+webencodings==0.5.1       # via html5lib
```

Comment:
This seems like a lot of dependencies just to produce the documentation. No changes needed though; it is what it is.

Reply:
If we're not publishing the documentation, perhaps we can remove these, and just have a normal docs folder? The code can be the API documentaiton.

Reply:
> If we're not publishing the documentation, perhaps we can remove these, and just have a normal docs folder?

We should eventually publish the documentation, so it's fine to leave those in. I don't think it hurts anything.